internal/sysfs/file_unix.go

```diff
@@ -19,3 +19,13 @@ func readFd(fd uintptr, buf []byte) (int, syscall.Errno) {
 	errno := platform.UnwrapOSError(err)
 	return n, errno
 }
+
+// writeFd exposes syscall.Write.
```

Comment:
this is a pretty cool way to ensure the logic is shared. nice!

Reply:
(actually I though you did this first...)

Reply:
uhh anyway thanks 😎